packages/mdc-ripple/package.json

```diff
@@ -15,7 +15,6 @@
   },
   "dependencies": {
     "@material/base": "^0.35.0",
-    "@material/theme": "^0.35.0",
-    "@material/animation": "^0.34.0"
```

Comment:
`@material/animation` is required for the `$mdc-animation-standard-curve-timing-function` variable in `_mixins.scss`

Reply:
Looks like I messed up with resetting the commits. This change is not supposed to be here. Reverting these changes.

Reply:
Reverted. PTAL.